pipeline-scripts/buildlib.groovy

```diff
@@ -1254,4 +1281,35 @@ def assertBuildPermitted(doozerOpts) {
     }
 }
 
+/**
+ * Run elliott find-builds and attach to given advisory.
+ * It looks for builds twice (rhel-7 and rhel-8) for OCP 4.y
+ * Side-effect: Advisory states are changed to "NEW FILES" in order to attach builds.
+ *
+ * @param String[] kinds: List of build kinds you want to find (e.g. ["rpm", "image"])
+ * @param String buildVersion: OCP build version (e.g. 4.2, 4.1, 3.11)
+ */
+def attachBuildsToAdvisory(String[] kinds, String buildVersion) {
+    def groupOpt = "-g openshift-${buildVersion}"
+    def isOCP4 = buildVersion.startsWith("4.")
+    def rhel8branchOpt = "--branch rhaos-${buildVersion}-rhel-8"
+
+    try {
+        if ("rpm" in kinds) {
+            elliott("${groupOpt} change-stage -s NEW_FILES --use-default-advisory rpm")
+            elliott("${groupOpt} find-builds -k rpm --use-default-advisory rpm")
+            if (isOCP4) {
+                elliott("${groupOpt} find-builds -k rpm --use-default-advisory rpm ${rhel8branchOpt}")
+            }
+        }
+        if ("image" in kinds) {
+            elliott("${groupOpt} change-stage -s NEW_FILES --use-default-advisory image")
+            elliott("${groupOpt} find-builds -k image --use-default-advisory image")
```

Comment:
hm yes it will ... what should we do here? use that `--between` to attach to main image and the rest to extras.
Not sure how we can get the `z` number though :disappointed:

Reply:
So now it is not required to get the builds into correct advisory within the scope of ART-722, although it is something nice to have.